chrome_extension/vendor/mooltipass/mcCombinations.js

```diff
@@ -140,6 +140,15 @@ var extendedPost = {
 	}
 }
 
+var extendedSubmitButton = {
+	'vodafone.de': function(form) {
+		return form.parent().parent().find('#loginBtn')
```

Comment:
in this case submit button is out of form container, need to let me look at code again to understand if we can unify this case